.github/workflows/pr-deploy.yaml

```diff
@@ -63,3 +70,30 @@ jobs:
         env:
           PR_CLUSTER_ID: ${{ secrets.PR_CLUSTER_ID }} # probably not _SECRET_, but best kept out of the YAML
 
+      - name: Add deploy comment
+        uses: peter-evans/create-or-update-comment@c9fcb64660bc90ec1cc535646af190c992007c32
+        with:
+          issue-number: ${{ github.event.pull_request.number }}
+          body: |
+            Change deployed and available at https://${{ steps.vars.outputs.fqdn_record }}
+
+            Comment with `!deploy-cleanup` to delete your deployment!
+        if: ${{ steps.check.outputs.has-permission && github.event.comment.body == '!deploy'}} 
+
+      # Only ran if the correct comment is used! We should have automatic cleanup on merge, but this allows us to manually
```

Comment:
If you need examples of destroy on merge there is something in the posthog-cloud-infra on merge of PRs to clean up test resources.